package.json

```diff
@@ -96,6 +96,7 @@
     "react": "^15.3.1",
     "react-addons-transition-group": "^15.3.0",
     "react-addons-update": "^15.3.1",
+    "react-copy-to-clipboard": "^5.0.0",
```

Comment:
I think you should remove it completely and follow the example in the StackOverflow answer. Since this is just for the Editor Tools, I don't think we have to worry so much about old browsers. It will only be used by the editors.

If you can remove the NPM package, implement it with pure JavaScript and just hide the copy button it if the browser doesn't support `Clipboard`, that would be great.